examples/IBFE/explicit/ex0/example.cpp

```diff
@@ -324,6 +326,10 @@ main(int argc, char* argv[])
         }
         std::unique_ptr<ExodusII_IO> exodus_io(uses_exodus ? new ExodusII_IO(mesh) : NULL);
 
+        // Check to see if this is a restarted run to append current exodus files
+        const bool from_restart = RestartManager::getManager()->isFromRestart();
+        exodus_io->append(from_restart);
```

Comment:
This might not work if libMesh is compiled without exodus support: we can get around this by checking `uses_exodus` before calling `ExodusII_IO::append()`.